app/components/Form/ApplicationForm.tsx

```diff
@@ -347,13 +349,37 @@ const ApplicationForm: React.FC<Props> = ({
       sectionName.toString()
     );
 
-    const newFormData = mergeFormSectionData(
+    let newFormData = mergeFormSectionData(
       jsonData,
       sectionName,
       calculatedSectionData,
       jsonSchema
     );
 
+    if (templateData) {
+      if (templateData.templateNumber === 1) {
+        newFormData = {
+          ...newFormData,
+          benefits: {
+            ...newFormData.benefits,
+            householdsImpactedIndigenous:
+              templateData.data.result.finalEligibleHouseholds,
+            numberOfHouseholds:
+              templateData.data.result.totalNumberHouseholdsImpacted,
+          },
+        };
+      } else if (templateData.templateNumber === 2) {
+        newFormData = {
+          ...newFormData,
+          budgetDetails: {
+            ...newFormData.budgetDetails,
+            totalEligibleCosts: templateData.data.result.totalEligibleCosts,
+            totalProjectCosts: templateData.data.result.totalProjectCosts,
```

Comment:
Nice work, this is working well except I noticed the `totalProjectCosts` wasn't being imported. I checked the excel and the data was there and being imported in the api. Tracked it down to this, looks like `totalProjectCosts` field is actual `totalProjectCost` in the `budgetDetails` page 🤦 

:facepalm:
--


<br class="Apple-interchange-newline">

:facepalm:
--


<br class="Apple-interchange-newline">

Reply:
🤦 good catch, thanks! Changing now